Lib/test/test_tokenize.py

```diff
@@ -3178,5 +3180,85 @@ def test_newline_at_the_end_of_buffer(self):
             run_test_script(file_name)
 
 
+class CommandLineTest(unittest.TestCase):
+    def setUp(self):
+        self.filename = tempfile.mktemp()
+        self.addCleanup(os_helper.unlink, self.filename)
+
+    @staticmethod
+    def text_normalize(string):
+        """Dedent *string* and strip it from its surrounding whitespaces.
+
+        This method is used by the other utility functions so that any
+        string to write or to match against can be freely indented.
+        """
+        def normalize_spaces(text):
+            return re.sub(r'\s+', ' ', text).strip()
+        return normalize_spaces(dedent(string)).strip()
```

Comment:
We don't need both these `.strip()`s?

Can we one-line it?

```suggestion
        return re.sub(r'\s+', ' ', dedent(string)).strip()
```

And does the `re.sub` mean we can remove `dedent`?

```suggestion
        return re.sub(r'\s+', ' ', string).strip()
```

Reply:
yes, you're right. thank you